.gitignore

```diff
@@ -16,3 +16,5 @@ __pycache__/
 # pytest coverage
 .coverage
 htmlcov/
+
+dev-requirements.txt

```

Comment:
Why is this required?

Reply:
CI generates this file and I wanted to ensure it never gets committed back to the repo